src/anonymous-credentials/schema.ts

```diff
@@ -982,6 +992,11 @@ export class CredentialSchema extends Versioned {
     return min >= 0 ? { type: this.POSITIVE_INT_TYPE } : { type: this.INT_TYPE, minimum: min };
   }
 
+  static parseDateType(node: object, parsingOpts: ISchemaParsingOpts, nodeName: string): object {
+    const min = parsingOpts.defaultMinimumInteger;
```

Comment:
Default minimum is negative but the numeric value of datetime won't ever be negative.  Are you planning to support some minimum hardocded date?
Secondly if the encoder is capable of converting datetime type to numeric minimum should not be numeric but a datetime.

Reply:
That's a good point. I was using a minimum placeholder in case it's not defined `new Date('0000-01-01')`, do you have any other suggestion? 

Actually, the timestamp for this placeholder minimum date is `-62167219200000` which is a negative number (not Unix timestamp)

Reply:
>Default minimum is negative but the numeric value of datetime won't ever be negative

this is a wrong assumption. we want to support all dates, not just post 1970

Reply:
I think we should support it, as it would be used for a birth date use case

Reply:
If you want to support dates older than 1970, then `-(Math.pow(2, 32) - 1)` is not a reasonable lower bound as that takes you about 50 days back only.

Reply:
agree, it wasnt so bad when unix epochs were used but we're now using js timestamps to be consistent. your input would be appreciated here on what a good lower bound would be, that would be supported by the encoder

Reply:
You tell me as you're dealing with the app layer :). So you will need older dates for the age of people, buildings,  ancient artifacts, etc so the earliest dates being around the 14th century should be sufficient? `-(Math.pow(2, 44) - 1)` works for that. Make it a separate constant in `DefaultSchemaParsingOpts` called  `defaultMinimumDate` or something.

Reply:
if there are no library limitations as to how low the number can be, then sure i'll come up with something - no problem

Reply:
i think that your suggestion is a reasonable default, go too far and we may have problems - for any other use case before that date they can supply their own minimum value. i do wonder how "future proof" it JS timestamps are in this case, but we wont experience that issue for many years i think